nvtabular/inference/triton/model_hugectr.py

```diff
@@ -83,16 +83,17 @@ def execute(self, requests: List[InferenceRequest]) -> List[InferenceResponse]:
                 output_tensors.append(Tensor("DES", np.array([0])))
 
             if "cats" in self.column_types:
+                cats_np = _convert_cudf2numpy(output_df[self.column_types["cats"]], np.uint32)
                 output_tensors.append(
                     Tensor(
                         "CATCOLUMN",
-                        _convert_cudf2numpy(output_df[self.column_types["cats"]], np.uint32),
+                        cats_np,
```

Comment:
Can you run black on this? its failing CI because of the formatting -

Reply:
Will do! Keep forgetting it.